src/openloco/window.h

```diff
@@ -27,7 +27,7 @@ namespace openloco::ui
     {
         widget_type type; // 0x00
         uint8_t colour;   // 0x01
-        int16_t left;     // 0x02
+        uint16_t left;    // 0x02
```

Comment:
I changed this so that the widget->left could be set to 0x8000. Which is then checked in drawTabs to see if the widget is disabled. But this causes the CI checks to fail due to `if (widget.left != -2)` in WIndowManager.cpp being always true. I will revert it and check for disabled widgets instead.